src/autocomplete.ts

```diff
@@ -1,21 +1,41 @@
 import type AutocompleteElement from './auto-complete-element'
 import debounce from './debounce'
 import {fragment} from './send'
+import getAnnouncementStringByEvent, {ScreenReaderAccouncementConfig} from './screen-reader-announcements'
 import Combobox from '@github/combobox-nav'
 
 export default class Autocomplete {
   container: AutocompleteElement
   input: HTMLInputElement
   results: HTMLElement
   combobox: Combobox
+  feedback: HTMLElement | null
+  autoselectEnabled: boolean
+  clientOptions: NodeListOf<HTMLElement> | null
 
   interactingWithList: boolean
 
-  constructor(container: AutocompleteElement, input: HTMLInputElement, results: HTMLElement) {
+  constructor(
+    container: AutocompleteElement,
+    input: HTMLInputElement,
+    results: HTMLElement,
+    autoselectEnabled?: boolean
+  ) {
     this.container = container
     this.input = input
     this.results = results
     this.combobox = new Combobox(input, results)
+    this.feedback = document.getElementById(`${this.results.id}-feedback`)
+    this.autoselectEnabled = !!autoselectEnabled
+
+    // check to see if there are any default options provided
+    this.clientOptions = results.querySelectorAll('[role=option]')
+
+    // make sure feedback has all required aria attributes
+    if (this.feedback) {
+      this.feedback?.setAttribute('aria-live', 'assertive')
+      this.feedback?.setAttribute('aria-atomic', 'true')
```

Comment:
Since we already do an `if` check, I think the `.?` would be redundant here.
```suggestion
      this.feedback.setAttribute('aria-live', 'assertive')
      this.feedback.setAttribute('aria-atomic', 'true')
```

Reply:
Legit 👍